examples/autocomplete/README.md

```diff
@@ -0,0 +1,58 @@
+# Autocomplete Examples
+
+Here you can find a few example implementations of the autocomplete functionality utilizing the Google Places API.
+
+## Examples
+
+We have three different implementations to demonstrate how to add autocomplete functionality to your application
+
+### 1) Google Maps Autocomplete Widget
+
+When using the [Google Maps Autocomplete widget][autocomplete-widget] you provide an HTML input element of your choice and Google handles all the rest. It will fetch predictions when the user types and it will get the details for a place when the user selects a prediction from the list.
+
+### 2) Custom Build
+
+When you need complete control over every aspect of your autocomplete you can choose to build your own by utilizing the [Autocomplete Service][autocomplete-service] for fetching query predictions and the [Places Service][place-details] for fetching the place details.
+
+When building you own you are completely free but also responsible for the user experience of the autocomplete. You are also responsible for handling the autocomplete session with a [Session token][session-token]. This can easily be overlooked and may lead to not expected surprises when it comes to billing.
```

Comment:
```suggestion
When building your own you are completely free but also responsible for the user experience of the autocomplete. You are also responsible for handling the autocomplete session with a [Session token][session-token]. This can easily be overlooked and may lead to not expected surprises when it comes to billing.
```